perf_benchmark.py

```diff
@@ -43,9 +43,14 @@
     if func_type[0] == "hypothesis":
         # read path from arguments
         path = configuration.get_schema_path(tada_configuration_dict)
+    level = tada_configuration_dict.get("level")
     # generate data
-    data = generate.generate_data(func_type, chosen_size, path)
+    data = generate.generate_data(func_type, chosen_size, level, path)
```

Comment:
`level` could be set as an optional argument with a default value of 1, so it does not introduce more complexity to the code after